ftrip.io.catalog-service/ftrip.io.catalog-service.contracts/ftrip.io.catalog-service.contracts.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>ftrip.io.catalog_service.contracts</RootNamespace>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+  </PropertyGroup>
+  
+  <ItemGroup Condition="'$(Configuration)'=='Release'">
+    <PackageReference Include="ftrip.io.framework" Version="2.0.0" />
```

Comment:
Can you please update to the latest version of the framework? You can find the latest versions here:
https://github.com/ftrip-io/framework